mds/enums.py

```diff
@@ -8,3 +8,12 @@
     ("unavailable", _("Unavailable")),
     ("removed", _("Removed")),
 ]
+DEVICE_TYPE_CHOICES = [
+    ("bike", _("Bike")),
```

Comment:
You need to manually execute `python manage.py makemessages` to update the translation files.